### PR TITLE
Updating NPM to v1.1.2

### PR DESCRIPTION
**Pull request by jaer-tsun**
Updating NPM to v1.1.2

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -201,7 +201,7 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
-1.1.1
```
Comment (xuto2) [inline vhdbuilder/packer/install-dependencies.sh:204]: is 1.1.1 not in use?

Comment (xuto2) [reply vhdbuilder/packer/install-dependencies.sh:204]: ok 1.1.1 was never in use. 
Can you at least put some description to the release? what is the fix?

Comment (jaer-tsun) [reply vhdbuilder/packer/install-dependencies.sh:204]: We didn't pull the latest AKS-E NPM tests so we missed a bug.

**Pull request merged**